assets/prototype/application/utilities/money/amountsMatch.js

```diff
@@ -1 +0,0 @@
-export const amountsMatch = (v1, v2) => parseFloat(v1) === parseFloat(v2);
```

Comment:
@tn3rb this seems to be a duplicate since there is already the `amountsMatch.ts` in place.
Let me know if removing this file is ok.